### PR TITLE
New Ruin: The Beach Town

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Adds a new beach ruin, the abandoned beachside town

dreamseeker_Ht2YcvyQbH
dreamseeker_KAB6kPSLrP
dreamseeker_8Xe7Cuq6NH
dreamseeker_SKJXeK9SOt
dreamseeker_6Ak0bNoVe5

The town is an mostly empty place formerly devoted to tourism and the beloved art of "chilling out". Facets of the life of its inhabitants before their disappearance included drinking, grilling, and swimming off the coast of their fairly large beach. Many interesting things happened on the boardwalk, and a landing pad was present to allow for small ships to dock inside the town. 

The loot list is sparse here. I intend for this to mostly be a setpiece for roleplay instead of a loot pinata. There's a good selection of hydroponics seeds and gear, 2 full bar kits, basic kitchen equipment, an autolathe, a few PDAs, a lotta wood, and a jukebox. Also donuts.

- [x] Ruin spawns, nothing is out of whack that shouldn't be.

## Why It's Good For The Game
Continues the trend of making planets more good by adding more content

## Changelog

:cl:
add: An oddly empty town has been spotted on beach planets in the area. Check it out spacers. 
add: Random donut spawners, never eat the same donut two days in a row!

/:cl:

### CODE REVIEW

**Comment by Latentish** [comment]
I'd like to see the areas. If you do that, and they aren't fucked, and the checks pass the gas, then we can look at TMing this.

File: code/game/area/areas/ruins/beachplanet.dm [hunk 1]
```diff
@@ -1 +1,24 @@
 //tropical ruins here
+
+//Beachside Town
```
Comment (Latentish) [inline code/game/area/areas/ruins/beachplanet.dm:4]: Again try to cut these down.

**Comment by Erikafox** [comment]
>I'd like to see the areas. If you do that, and they aren't fucked

top screenshot

File: _maps/map_catalogue.txt [hunk 2]
```diff
@@ -480,6 +480,9 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 18)(y = 14)(z = 1)
 		Tags = "Medium Combot Challenge", "Medium Loot", "Liveable"
 
+		File Name = "_maps\RandomRuins\BeachRuins\beach_ocean_town.dmm"
+		Size = (x = 80)(y = 83)(z = 1)
+		Tags = "No Combat", "Minor loot"
```
Comment (Latentish) [inline _maps/map_catalogue.txt]: needs an atmosphere tag

**Review by Latentish — DISMISSED** [review]

**Comment by Zevotech** [comment]
round 1272 where this was on a planet the sandstone room with the assistant statue had space tiles

**Comment by AverageUser67** [comment]
can you fix the sand and also glass tiles that show space

**Comment by Erikafox** [comment]
No sorry that'd be too much work

**Comment by Erikafox** [comment]
@Latentish please lift your requested changes, they've been addressed for a while.

**Comment by Latentish** [comment]
Marg

**Pull request merged**